lobster/tools/codebeamer/codebeamer.py

```diff
@@ -210,17 +253,50 @@ def import_tagged(mh, cb_config, items_to_import):
     return rv
 
 
+def ensure_array_of_strings(instance):
+    if isinstance(instance, list) and all(isinstance(item, str) for item in instance):
+        return instance
+    else:
+        return [str(instance)]
+
+
+def parse_cb_config_file(file_name):
+    assert isinstance(file_name, str)
+    assert os.path.isfile(file_name)
+
+    with open(file_name, "r") as file:
+        data = json.loads(file.read())
+
+        provided_config_keys = set(data.keys())
+        supported_references = set(SUPPORTED_REFERENCES)
+
+        if not provided_config_keys.issubset(supported_references):
+            raise Exception("The provided references are not supported! "
+                            "supported referenes: '%s'" % ', '.join(SUPPORTED_REFERENCES))
+
+        json_config = {}
+        for key, value in data.items():
+            json_config[key] = ensure_array_of_strings(value)
+        return json_config
+
+
 def main():
     ap = argparse.ArgumentParser()
 
     modes = ap.add_mutually_exclusive_group(required=True)
     modes.add_argument("--import-tagged",
                        metavar="LOBSTER_FILE",
                        default=None)
+
     modes.add_argument("--import-query",
                        metavar="CB_QUERY_ID",
                        default=None)
 
+    ap.add_argument("--config-file",
```

Comment:
Please name it just `--config` to keep it short, and to keep it aligned with all the other lobster tools.

Reply:
Sure

Just for information: 

- "trlc" also uses "--config-file" as an argument
- "report" uses "--lobster-config" as an argument